pynicotine/gtkgui/chatrooms.py

```diff
@@ -658,14 +658,14 @@ def read_room_logs_finished(self):
 
         self.activity_view.auto_scroll = self.chat_view.auto_scroll = True
 
-    def read_room_logs(self):
+    def prepend_old_messages(self):
 
-        self.chat_view.append_log_lines(
-            folder_path=log.room_folder_path,
-            basename=self.room,
-            num_lines=config.sections["logging"]["readroomlines"],
-            timestamp_format=config.sections["logging"]["rooms_timestamp"]
-        )
+        log_lines = self._read_old_messages()
+
+        self.chat_view.append_log_lines(log_lines, login_username=config.sections["server"]["login"])
+
+    def _read_old_messages(self):
+        return log.read_log(log.room_folder_path, self.room, config.sections["logging"]["readroomlines"])
```

Comment:
Could you pass the arguments as keyword arguments instead? Would be easier to see what they do at a glance.